src/Components/Connect.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import './../Styles/index.css'
+import TG from "./../Image/teleg.png"
+import VK from "./../Image/vk.png"
+import Git from "./../Image/git.png"
```

Comment:
Try to name Icons and icons files in one style. 
I usually do:
icon-"icon-name" or picture-"picture-name"